### ISSUE TITLE
Logger modifications: changed levels and verbosity flags

### ISSUE THREAD

**Issue by nsoblath** [project8/scarab#67]
From driplineorg/dripline#26:

Change the logger levels:
Trace = 0
Debug= 10
Info = 20
Prog = 25
Warn = 30
Fatal = 40

Default logger level to be 25.

Add `-v` flags (can have multiple) to the CLI.  Each subtracts 10 from the verbosity threshold.

Add `-q` flags (can have multiple) to the CLI.  Each adds 10 to the verbosity threshold.

Actually, now that I look at it, I need to fix the setting of the verbosity in `main_app::pre_callback()`.  Currently it only sets the verbosity after the stage-1 configuration.  It needs to reset it after the 4th stage.

**Comment by nsoblath** [comment]
On a more technical note, the plan for implementation is:

In `logger`:
```
enum class ELevel {
    eTrace = 0,
    eDebug = 10,
    eInfo = 20,
    eProg = 25,
    eWarn = 30,
    eError = 40,
    eFatal = 50
};
```

and in `main_app`:
```
public:
    unsigned get_global_verbosity() const;
    void set_global_verbosity( unsigned );

    void offset_global_verbosity( int );

private:
    static vector< logger::ELevel > f_verbosity_levels;
    vector< logger::ELevel >::iterator f_global_verbosity;
```

**Comment by nsoblath** [comment]
Released in v2.11.0
